src/components/EventCalendar/EventCalendarListPanel.tsx

```diff
@@ -125,7 +128,18 @@ const EventCalendarListPanel = ({ initialRequiredLabelIds, initialOptionalLabelI
         },
     })
 
+    const transformEventsByMonth = (events: Array<CalendarEventDataFragment>): EventsByMonth => {
+        const monthsInYear = 12
+        const eventsByMonth = generateTwoDimensionalArray(monthsInYear, 0)
+        events?.forEach((event: CalendarEventDataFragment) => {
+            const monthOfEvent = parseDateTime(event.from).getMonth()
```

Comment:
Celé tohle má jednu zásadní chybu a to že se do sebe budou (mohouú motat události ve stejném měsíci z různých let.